AmplifyPlugins/DataStore/Tests/AWSDataStorePluginTests/Migration/SQLiteMutationSyncMetadataMigrationDelegateTests.swift

```diff
@@ -75,14 +75,14 @@ class SQLiteMutationSyncMetadataMigrationDelegateTests: MutationSyncMetadataMigr
         try setUpAllModels()
         let restaurant = Restaurant(restaurantName: "name")
         save(restaurant)
-        let metadata = MutationSyncMetadata(id: restaurant.id, deleted: false, lastChangedAt: 1, version: 1)
+        let metadata = MutationSyncMetadata(modelId: restaurant.id, modelName: Restaurant.modelName, deleted: false, lastChangedAt: 1, version: 1)
```

Comment:
i don't think we should change this test, despite it's marked as deprecated, the old init is allowing us to set up the migration script testing with old day and verifying that the migration works